### PR TITLE
[bitnami/suitecrm] Fix for smtpPassword and SUITECRM_SMTP_PORT

### PR DESCRIPTION
**Pull request by malagant**
**Description of the change**

When deploying bitnami/suitecrm the pod complains about missing smtp password and port.

**Benefits**

With this fix suitecrm will deploy correctly when using smtp settings.

**Possible drawbacks**

None.

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/suitecrm/templates/deployment.yaml [hunk 1]
```diff
@@ -187,11 +187,11 @@ spec:
               value: {{ .Values.suitecrmNotifyAddress | quote }}
             - name: SUITECRM_SMTP_HOST
               value: {{ .Values.suitecrmSmtpHost | quote }}
-            - name: SUITECRM_SMTP_PORT_NUMBER
+            - name: SUITECRM_SMTP_PORT
```
Comment (juan131) [inline bitnami/suitecrm/templates/deployment.yaml:190]: The env. variable used to configure SMTP is actually `SUITECRM_SMTP_PORT_NUMBER`, see:

- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/libsuitecrm.sh#L303

That said, both are aliases so it would work using both formats. In any case, please revert this change

File: bitnami/suitecrm/templates/deployment.yaml [hunk 2]
```diff
@@ -187,11 +187,11 @@ spec:
...
               value: {{ .Values.suitecrmSmtpPort | quote }}
             - name: SUITECRM_SMTP_USER
               value: {{ .Values.suitecrmSmtpUser | quote }}
-            {{- if .Values.smtpPassword }}
+            {{- if .Values.suitecrmSmtpPassword }}
```
Comment (juan131) [inline bitnami/suitecrm/templates/deployment.yaml:194]: Nice catch!

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for this PR! Please check my comments.

File: bitnami/suitecrm/templates/deployment.yaml (hunk 1 above)
Comment (malagant) [reply bitnami/suitecrm/templates/deployment.yaml:190]: Sorry, but when I open the link I only see SUITECRM_SMTP_PORT. Am I missing something?

Comment (juan131) [reply bitnami/suitecrm/templates/deployment.yaml:190]: Sorry, I shared the wrong link... Here we're using `SUITECRM_SMTP_PORT_NUMBER`:
- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/libsuitecrm.sh#L74
It seems there's some inconsistency around this, I'll fix it in the container image. 
Here you can find they're actually aliases:
- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/suitecrm-env.sh#L86
But we should use `SUITECRM_SMTP_PORT_NUMBER` for consistency with other Bitnami containers.

Comment (malagant) [reply bitnami/suitecrm/templates/deployment.yaml]: Thanks for the clarification. I Reverted the change. Thank you again.

**Review by juan131 — APPROVED** [review]
LGTM
